.github/workflows/lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Lint
+
+on:
+  pull_request
+
+jobs:
+  flake8_py3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7.4
+          architecture: x64
+      - name: Checkout PyTorch
+        uses: actions/checkout@master
+      - name: Run flake8
+        run: |
+          set -eux
+          pip install flake8
+          flake8 --exit-zero > ${GITHUB_WORKSPACE}/flake8_output.txt
+      - name: Add annotations
+        uses: suo/add-annotations-github-action@master
```

Comment:
Can we put this repo in pytorch org, managed by OSS, pretty please? :) Whitelist your new repo https://our.internmc.facebook.com/intern/diff/D16811608/